searx/settings.yml

```diff
@@ -1277,6 +1277,13 @@ engines:
       require_api_key: false
       results: HTML
 
+  # - name: marginalia
+  #   engine: marginalia
+  #   # contact for an API key: contact@marginalia-search.com
+  #   # rate-limited API key, for test & debug: public
```

Comment:
> But maybe the python module should include code that prevents the public API key from being used at all? Bassicly give an error instead of allowing it to be used?
>
I'd just remove this line here so that people contact for an API key instead.